File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P06_TheatrePromotions/P06_TheatrePromotions.cs
using System;

namespace P06_TheatrePromotions
{
    class P06_TheatrePromotions
    {
        static void Main(string[] args)
        {       
            string day = Console.ReadLine();
            int age = int.Parse(Console.ReadLine());

            bool age1 = (age >= 0 && age <= 18);
            bool age2 = (age > 18 && age <= 64);
            bool age3 = (age > 64 && age <= 122);

            switch (day)
            {
                case "Weekday":
                    if (age1)
                    {
                        Console.WriteLine("12$");
                    }
                    else if (age2)
                    {
                        Console.WriteLine("18$");

                    }
                    else if (age3)
                    {
                        Console.WriteLine("12$");
                    }
                    else
                    {
                        Console.WriteLine("Error!");
                    }
                    break;
                case "Weekend":
                    if (age1)
                    {
                        Console.WriteLine("15$");
                    }
                    else if (age2)
                    {
                        Console.WriteLine("20$");

                    }
                    else if (age3)
                    {
                        Console.WriteLine("15$");
                    }
                    else
                    {
                        Console.WriteLine("Error!");
                    }
                    break;
                case "Holiday":
                    if (age1)
                    {
                        Console.WriteLine("5$");
                    }
                    else if (age2)
                    {
                        Console.WriteLine("12$");

                    }
                    else if (age3)
                    {
                        Console.WriteLine("10$");
                    }
                    else
                    {
                        Console.WriteLine("Error!");
                    }
                    break;
                default:
                    Console.WriteLine("Error!");
                    break;
            }
        }
    }
}

File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P06_TheatrePromotions.java
import java.util.Locale;
import java.util.Scanner;
public class P06_TheatrePromotions {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);
        String type = console.nextLine().toLowerCase(Locale.ROOT);
        int age = Integer.parseInt(console.nextLine());
        boolean age1 = (age >= 0) && (age <= 18);
        boolean age2 = (age > 18) && (age <= 64);
        boolean age3 = (age > 64) && (age <= 122);

        switch (type) {
            case "weekday":
                if(age1) {
                    System.out.println("12$");
                } else if(age2) {
                    System.out.println("18$");
                } else if(age3) {
                    System.out.println("12$");
                } else {
                    System.out.println("Error!");
                }
                break;
            case "weekend":
                if(age1) {
                    System.out.println("15$");
                } else if(age2) {
                    System.out.println("20$");
                } else if(age3) {
                    System.out.println("15$");
                } else {
                    System.out.println("Error!");
                }
                break;
            case "holiday":
                if(age1) {
                    System.out.println("5$");
                } else if(age2) {
                    System.out.println("12$");
                } else if(age3) {
                    System.out.println("10$");
                } else {
                    System.out.println("Error!");
                }
                break;
            default:
                System.out.println("Error!");
                break;
        }
    }
}
